src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -114,8 +114,12 @@ const ColonyHome = ({
 
   const { colony } = data;
 
-  // Eventually this has to be in the proper domain. There's probably going to be a different UI for that
-  const canCreateTask = canAdminister(currentDomainUserRoles);
+  /*
+   * Eventually this has to be in the proper domain. There's probably going to be a different UI for that
+   * Also prevent users w/o a claimed profile from creating tasks
```

Comment:
This sounds like we would like to do that in the future. I don't think this comment is necessary (right now it's more confusing than helpful IMHO :) )

Reply:
Ok, I'll remove it then